InvenTree/common/models.py

```diff
@@ -1628,6 +1629,15 @@ def requires_restart(self):
             return False
 
 
+def label_printer_options():
+    """Build a list of available label printer options."""
+    printers = [('', _('no printer (Export to PDF)'))]
```

Comment:
Small thing, please capitalize "No Printer"

Reply:
good catch - that would be terribly inconsistent :facepalm: - fixed it!